kratos/processes/levelset_convection_process.h

```diff
@@ -893,7 +898,7 @@ class LevelSetConvectionProcess
                 "time_integration_theta" : 0.5
             })");
         } else {
-            KRATOS_ERROR << "Default parameters are not implemented for the specified \'" << ElementType << "\' element. Available options are \n\t- \'levelset_convection_supg\'\n\t- \'levelset_convection_algebraic_stabilization\'" << std::endl;
+            default_parameters = Parameters();
```

Comment:
I'd throw a warning in here and then return the empty `default_parameters` declared above.